packages/core/test/discovery.test.js

```diff
@@ -837,15 +838,28 @@ describe('Discovery', () => {
         '(?<stack>(.|\n)*)$'
       ].join('\n')));
     });
+
+    it('shows a warning when idle wait timeout is set over 60000ms', async () => {
+      process.env.PERCY_NETWORK_IDLE_WAIT_TIMEOUT = 80000;
+
+      await percy.snapshot({
+        name: 'test idle',
+        url: 'http://localhost:8000'
+      });
+
+      expect(logger.stderr).toContain(jasmine.stringMatching(
```

Comment:
can you add expectation with for this line as well?

https://github.com/percy/cli/blob/c12668d70252186f061399f00f4f2aebbcc06d8c/packages/core/src/network.js#L66

Reply:
I can add it, but its not related to current test though, this line is related to networkIdleTimeout